packages/devtools_app/lib/src/flutter/device_dialog.dart

```diff
@@ -0,0 +1,265 @@
+          '${flutterVersion.version} / ${flutterVersion.channel}';
+      versions['Framework / Engine'] = '${flutterVersion.frameworkRevision} / '
+          '${flutterVersion.engineRevision}';
+    }
+
+    return AlertDialog(
+      actions: [
+        if (connectedApp.isRunningOnDartVM) _ViewVMFlagsButton(),
+        DialogCloseButton(),
+      ],
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          ...headerInColumn(theme.textTheme, 'Device Info'),
+          for (var name in versions.keys)
+            Padding(
+              padding: const EdgeInsets.only(bottom: denseRowSpacing),
+              child: Row(
+                children: [
+                  Text('$name: ', style: boldText),
+                  Text(versions[name], style: theme.subtleTextStyle),
+                ],
+              ),
+            ),
+        ],
+      ),
+    );
+  }
+}
+
+class _ViewVMFlagsButton extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FlatButton(
+      onPressed: () {
+        Navigator.of(context, rootNavigator: true).pop('dialog');
+
+        unawaited(showDialog(
+          context: context,
+          builder: (context) => VMFlagsDialog(),
+        ));
+      },
+      child: Text('View VM Flags...'.toUpperCase()),
+    );
+  }
+}
+
+class VMFlagsDialog extends StatefulWidget {
+  @override
+  _VMFlagsDialogState createState() => _VMFlagsDialogState();
+}
+
+class _VMFlagsDialogState extends State<VMFlagsDialog> with AutoDisposeMixin {
+  InfoController infoController;
+  TextEditingController filterController;
+
+  List<_DialogFlag> flags = [];
+  List<_DialogFlag> filteredFlags = [];
+
+  @override
+  void initState() {
+    super.initState();
+
+    infoController = InfoController();
+
+    filterController = TextEditingController();
+    filterController.addListener(() {
+      setState(() {
+        _refilter();
+      });
+    });
+
+    _updateFromController();
+    addAutoDisposeListener(infoController.flagListNotifier, () {
+      setState(() {
+        _updateFromController();
+      });
+    });
+  }
+
+  void _updateFromController() {
+    flags = infoController.flagListNotifier.value.flags
+        .map((flag) => _DialogFlag(flag))
+        .toList();
+    _refilter();
+  }
+
+  void _refilter() {
+    final filter = filterController.text.trim().toLowerCase();
+
+    if (filter.isEmpty) {
+      filteredFlags = flags;
+    } else {
+      filteredFlags =
+          flags.where((flag) => flag.filterText.contains(filter)).toList();
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
+    return AlertDialog(
+      actions: [
+        DialogCloseButton(),
+      ],
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
```

Comment:
Since there are only two items you could do MainAxisAlignment.spaceBetween. or out the container In the expanded and align it to the right

Reply:
This is a bit easier for be to read the code - to look at it and know how it'll lay out. Happy to have someone change it in the future however.